src/layouts/HomeCards/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { describe, test, expect, vi, Mock } from 'vitest';
+import { render } from '@testing-library/react';
+import { HomeCards } from '.';
+import { fetchData, fetchedData } from '../../utils/requests';
+import { LoadingIcon } from '../../assets/LoadingIcon';
+import { Provider } from 'react-redux';
+import store from '../../store/store';
+
+global.fetch = vi.fn(() =>
+  Promise.resolve({
+    json: () => Promise.resolve(fetchedData),
+  })
+) as Mock;
+
+describe('Home Cards', () => {
+  test('makes a GET request', async () => {
+    const todoList = await fetchData(`https://api.jikan.moe/v4/anime?page=1`);
+    expect(fetch).toHaveBeenCalledWith('https://api.jikan.moe/v4/anime?page=1');
```

Comment:
Why there are no tests with actual RTK calling the API? Where are cases where it returns errors?

Not much of a test if you fetchData basically does nothing:

```
export const fetchData = async (link: string) => {
  try {
    return await fetch(link).then((res) => res.json());
  } catch (err) {
    console.log(err);
  }
};
```